sdk/communication/azure-communication-sms/azure/communication/sms/_generated/aio/_configuration.py

```diff
@@ -33,7 +33,7 @@ def __init__(
         super(AzureCommunicationSMSServiceConfiguration, self).__init__(**kwargs)
 
         self.endpoint = endpoint
-        self.api_version = "2020-07-20-preview1"
+        self.api_version = "2021-03-07"
```

Comment:
Same comment as above

Reply:
We discussed this offline and decided to leave it alone.